runners.go

```diff
@@ -32,16 +33,17 @@ type RunnersService struct {
 
 // Runner represents a GitLab CI Runner.
 //
-// GitLab API docs: https://docs.gitlab.com/ce/api/runners.html
+// GitLab API docs: https://docs.gitlab.com/ee/api/runners.html
```

Comment:
Why are you updating the URL? I prefer to reference the `ce` docs, unless there are fields or methods in there that are only supported by `ee`.

Reply:
Think I missed that in the other PR, can you refert that?

Reply:
sure thing